images/cross-cloud/sles-sap/byos/15-sp4-hardened/image.yaml

```diff
@@ -2,11 +2,12 @@ include-paths:
   - sle15/sp1
   - sle15/sp2
   - sle15/sp3
-  - hardened/sle15/sp3
+  - sle15/sp4
+  - hardened/sle15/sp4
 image:
   _attributes:
     schemaversion: "7.2"
-    name: SLES15-SP3-SAP-BYOS-Hardened
-    displayname: SLES15-SP3-SAP-BYOS-Hardened
+    name: SLES15-SP4-SAP-BYOS-Hardened
+    displayname: SLES15-SP4-SAP-BYOS-Hardened
```

Comment:
Name-flip please "....Hardened-BYOS" instead of "....BYOS-Hardened". The flip provides consistency with other images where we also have "build-flavor" ahead of "billing-flavor" in the image name.